### DISCUSSION TITLE
Need Tooltip throughout the height of the Marker

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#4282] [Q&A]
Hi, 
I need the marker to be pickable throughout the height of the marker,
This is the sandbox: https://www.itwinjs.org/sandboxes/MyPOC/Leese-the-bronze-falcon-744
currently its pickable at the bottom only

**Comment by pmconne** [comment]
Your code:
```
 public override addMarker(context: DecorateContext) {
  super.addMarker(context);
  const builder = context.createGraphicBuilder(GraphicType.WorldDecoration);
  builder.setSymbology(this._color, this._color, 1);
  builder.addSolidPrimitive(this.shape as Cone);
  context.addDecorationFromBuilder(builder);
 }
```

Everything after `super.addMarker(context)` has nothing to do with your Marker. Markers are 2d HTMLElements. You are creating a [3d WebGL graphic](https://www.itwinjs.org/reference/core-frontend/rendering/rendergraphic/).
